src/Compilers/CSharp/Test/Semantic/Semantics/OperatorTests.cs

```diff
@@ -424,7 +467,123 @@ static void Main()
 }";
             string output = @"bcfgjklln";
 
-            CompileAndVerify(source: source, expectedOutput: output);
+            CompileAndVerify(source: source, expectedOutput: output)
+                .VerifyOperationTree(@"
```

Comment:
We should be able to do verification off of a compilation instance. Calling CompileAndVerify is not an option for error scenarios, but we should test them too.